events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+                if (correctCount === 1) {
+                    prizeCalculator("game over");
+                } else {
+                    prizeCalculator("increase pot");
+                }
+            }, 1500);
+        } 
+        Entities.callEntityServerMethod(avatarCounter, "textUpdate", [count, true]);
+    }
+       
+    function prizeCalculator(gameState) {
+        var count = usersInZone(gameZoneProperties);
+        switch (gameState) {
+            case "new game":
+                if ( count <= MIN_PLAYERS ) {
+                    prizeMoney = MIN_PRIZE;
+                } else {
+                    prizeMoney = count * HFC_INCREMENT; 
+                }
+                break;
+            case "everyone wrong":                
+                prizeMoney *= HFC_HALVER;
+                if (prizeMoney <= MIN_PRIZE) { 
+                    prizeMoney = MIN_PRIZE;
+                }
+                Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['POT_DECREASE_SFX']);
+                Entities.callEntityServerMethod(gameZoneProperties.id, "loseCoins");
+                Entities.callEntityServerMethod(gameZoneProperties.id, "halfHFC");
+                break;
+            case "increase pot":
+                prizeMoney += HFC_INCREMENT;
+                Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['POT_INCREASE_SFX']);
+                Entities.callEntityServerMethod(gameZoneProperties.id, "winCoins");
+                Entities.callEntityServerMethod(gameZoneProperties.id, "plusHFC");
+                break;
+            case "game over":
+                prizeMoney += HFC_INCREMENT;
+                Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['WINNER_MUSIC']);    
+                Entities.callEntityServerMethod(gameZoneProperties.id, "startConfetti");              
+                if (prizeMoney >= 300 && winnerID !== MyAvatar.sessionUUID) {
+                    var winnerBanner = AvatarManager.getAvatar(winnerID).sessionDisplayName + " wins the game!";
+                    Entities.callEntityServerMethod(questionText, "textUpdate", [winnerBanner, true]);
+                    Users.requestUsernameFromID(winnerID);
+                }                
+                break;
+        }   
+    }
+
+    function sendInput(winningUserName) {     
+        if (prizeMoney >= 300 && winnerID !== MyAvatar.sessionUUID && winningUserName !== AccountServices.username) {
+            var hostPayout = prizeMoney * HOST_PERCENTAGE;    
+            var paramString = encodeURLParams({
+                date: new Date(),
+                triviaMasterUserName: AccountServices.username,
+                triviaMasterPayout: hostPayout,
+                winnerUserName: winningUserName,
+                winnings: prizeMoney,
+                senderID: AccountServices.username
+            });
+            var request = new XMLHttpRequest();
```

Comment:
any reason you aren't using the `request` module?

Reply:
Another "Becky wrote this"... I will definitely be looking more at modules going forward.